handbook/company/pricing-features-table.yml

```
@@ -929,7 +929,7 @@
 #  ╩ ╩╩ ╩╩═╝╚╩╝╩ ╩╩╚═╚═╝  ═╩╝╚═╝ ╩ ╚═╝╚═╝ ╩ ╩╚═╝╝╚╝  └─ ╩ ╩ ╩╩╚═╩ ╩─┘
 - industryName: Malware detection (YARA/custom IoCs)  # TODO: consider: technically more than YARA, consider generalizing this and including the concept of comparing known binary hashes and other IoCs (either via live query or in the data lake to compare threat intel feed)
   friendlyName: Scan files for zero days and malware signatures
-  description: Use YARA signatures to report and trigger automations when zero days, malware, or unexpected files are detected on a host.
+  description: Use YARA signatures to report and trigger automations when zero days, malware, or unexpected files are detected on a host.  YARA rules can be deployed remotely and privately.
```

```suggestion
  description: Deploy YARA signatures (rules) to report and trigger automations when zero days, malware, or unexpected files are detected on a host. 
```

Small copy tweak. I think users expect to be able to deploy "remotely and privately" (the new way) even though you can technically deploy rules publicly (old way). I think let's make the new way as the best practice.

---

Gotcha.  I was thinking this was a selling point since it seems that only Fleet supports it right now, so seemed like it was worth explicitly pointing out.